trunk/SUAVE/Plots/Geometry_Plots/plot_airfoil.py

```diff
@@ -33,20 +33,38 @@ def plot_airfoil(airfoil_names,  line_color = 'k-', save_figure = False, save_fi
     """
     # get airfoil coordinate geometry     
     airfoil_data = import_airfoil_geometry(airfoil_names)       
-
-    for i in range(len(airfoil_names)):
-        # separate x and y coordinates 
-        airfoil_x  = airfoil_data.x_coordinates[i] 
-        airfoil_y  = airfoil_data.y_coordinates[i]    
-
-        name = save_filename + '_' + str(i)
+    
+    if overlay:
```

Comment:
I'm not sure if was from this script or the one before. But it spit out a bunch of blank plots.

Reply:
hmm.. on my end it gives the one plot of overlayed airfoil geometries and then the 4 individual airfoil plots (5 plots total). Let me know if this is still happening for you with the new changes